miner/worker.go

```diff
@@ -1009,6 +1009,9 @@ func (w *worker) commit(uncles []*types.Header, interval func(), update bool, st
 	if err != nil {
 		return err
 	}
+
+	s.CorrectAccountsRoot(w.chain.CurrentBlock().Root())
```

Comment:
If the miner doesn't enable the pipe commit, there is no need to correct the root?